util/tools/translator.ts

```diff
@@ -11,6 +11,17 @@ import ora, { Ora } from "ora";
  * @author Bas950 <me@bas950.com>
  */
 
+/**
+ * USING THE TRANSLATOR TOOL
+ *
+ * - Open up a terminal and run yarn translator
+ *	- When prompted, select the Language Code for the language you are translated (e.g `ga_IE` for Irish)
+ *	- Once selected. The script will first cache all presences metadata files, it will then go through each one and remove the files containing the language you inputted from the cache.
+ *	- You will then be prompted to select a mode, afterwards you might get asked some aditional information to finalize your filter.
```

Comment:
You should explain what a mode is, what modes there are, and the differences between them.